tools/crm_resource_runtime.c

```diff
@@ -2108,7 +2108,7 @@ wait_till_stable(pcmk__output_t *out, guint timeout_ms, cib_t * cib)
     if (timeout_ms == 0) {
         expire_time += WAIT_DEFAULT_TIMEOUT_S;
     } else {
-        expire_time += pcmk__timeout_ms2s(timeout_ms + 999);
+        expire_time += pcmk__timeout_ms2s(timeout_ms);
```

Comment:
ACK on the patch itself, but I find it a little confusing that we let the user specify seconds/milliseconds/whatever on the command line, and then just convert it to seconds and lose whatever extra precision the user specified.

Reply:
I considered capping it -- add 999 only if `timeout_ms <= GUINT_MAX - 999`, to prevent overflow and preserve behavior. That's totally doable. I just feel like that's unnecessarily confusing. This function is only being run from a CLI invocation of `crm_resource`. I'm struggling to imagine a use case where rounding down to the nearest second would make any difference. Like, say a user specifies 1499 ms and the timeout occurs in 1 second. It's odd, but... is anyone gonna notice? Just use a bigger timeout.

We could leave it as-is, cap the value as suggested above, or rework the code to use higher-resolution time (like `clock_gettime()`) so that we can "honor" the value that the user specified on the command line.